src/components/PhoneInput/index.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import PhoneDetailsInput from '.'
+
+const countries = [
+  {
+    code: 1441,
+    name: 'Bermuda',
+    value: 'bm',
+  },
+  {
+    code: 975,
+    name: 'Bolivia',
+    value: 'bo',
+  },
+  {
+    code: 591,
+    name: 'Bhutan',
+    value: 'bt',
+  },
+  {
+    code: 31,
+    name: 'The Netherlands',
+    value: 'nl',
+  },
+  {
+    code: 40,
+    name: 'Romania',
+    value: 'ro',
+  },
+]
+
+storiesOf('PhoneInput', module)
+  .add('basic', () => (
+    <PhoneDetailsInput
+      id="phonenumber"
+      label="Phone number"
+      countries={countries}
+      onChange={e => console.log(e)}
```

Comment:
Just to make it more clear in the stories, use both vars here with proper names.

`onChange={(country, number) => console.log(country, number)}` for example.

When testing I thought it was a bug that the number wasn't showing after change, but on reading the code I saw it was the second param (=